docs/book/src/reference/completion.md

```diff
@@ -0,0 +1,29 @@
+# Enable shell auto completion
+
+## Prerequisites
+
+- Bash 4.1+/zsh 
+- bash-completion/zsh-completions v2
+
```

Comment:
IMO it should be : 

> ## Bash on Linux
> 
> //Add the steps
> 
> ## Bash on Mac
> 
> // Add the steps
> 
> ## Zsh 
> 
> // Add the steps

Reply:
We should avoid discussing installation steps on different systems, since "linux" has various distros (read: package managers).

Listing the prerequisite packages and versions delegates installation (which is relatively simple in this case) to the user. By doing so we also avoid doc drift in case prereqs change.

Reply:
@estroz @camilamacedo86 
As I could test out on macOS, I can list out steps for the same.
Any idea on how do folks go about for other distributions? 
Can I leave a TODO note for others such as ""linux", and "zsh"?

Reply:
The problem here is that I am not sure if the same steps will work for both (Linux or Mac).

Regards the pre-requirement for bash it is the same for kubectl and for both (Linux or Mac) its docs is addressing the same repo. There has the info over how to install in each SO. 

So, IMO we can move forward with:

# Enabling shell autocompletion

(introduction)

## Bash

(preretirement note)

(steps)

Then, we can improve it in the future.

Reply:
@camilamacedo86 
So, no changes for now. Did I understand correctly?